### PR TITLE
Update logging information for macOS

### PR DESCRIPTION
**Pull request by sdwheeler**
# PR Summary

Update logging information for macOS

- Fixes [AB#303911](https://dev.azure.com/msft-skilling/cebd7ef5-4282-448b-9701-88c8637581b7/_workitems/edit/303911)

## PR Checklist

- [x] **Descriptive Title:** This PR's title is a synopsis of the changes it proposes.
- [x] **Summary:** This PR's summary describes the scope and intent of the change.
- [x] **Contributor's Guide:** I have read the [contributors guide][contrib].
- [x] **Style:** This PR adheres to the [style guide][style].

[contrib]: https://learn.microsoft.com/powershell/scripting/community/contributing/overview
[style]: https://learn.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide

### CODE REVIEW

File: reference/7.2/Microsoft.PowerShell.Core/About/about_Logging_Non-Windows.md [hunk 1]
```diff
@@ -279,52 +280,20 @@ log file named `powershell.log`.
...
+events. The PowerShell log events don't appear in the **Console** application
+on macOS. Console app is designed for the older _syslog-based_ logging that
+predates the unified logging system.
 
 ### Viewing PowerShell log data from the command line on macOS
 
 To view PowerShell log data from a command line on macOS, use the `log` command
 in the **Terminal** or other shell host application. These commands can be run
-from **PowerShell**, **Z shell** (**Zsh**), or **Bash**.
+from **PowerShell**, **ZShell**, or **Bash**.
```
Comment (mikefrobbins) [inline reference/7.2/Microsoft.PowerShell.Core/About/about_Logging_Non-Windows.md:296]: According to Apple, it's `zsh` (Z shell) [support.apple.com/102360](https://support.apple.com/102360).

File: reference/7.2/Microsoft.PowerShell.Core/About/about_Logging_Non-Windows.md [hunk 2]
```diff
@@ -333,9 +302,38 @@ running, the **process** parameter also accepts a process ID as its value. The
...
+```powershell
+log show --start "2022-04-05 09:00:00" --predicate "subsystem == 'com.microsoft.powershell'"
+```
+
+For more information, run `log show --help` to view the help for the `log show`
+command.
+
+You can also output the log data in JSON format, which allows you to convert
+the event data to PowerShell objects. The following example outputs the events
+in JSON format.The `ConvertFrom-Json` cmdlet is used to convert the JSON data
```
Comment (mikefrobbins) [inline reference/7.2/Microsoft.PowerShell.Core/About/about_Logging_Non-Windows.md:324]:
```suggestion
in JSON format. The `ConvertFrom-Json` cmdlet is used to convert the JSON data
```

File: reference/7.4/Microsoft.PowerShell.Core/About/about_Logging_Non-Windows.md [hunk 3]
```diff
@@ -333,9 +302,38 @@ running, the **process** parameter also accepts a process ID as its value. The
...
+```powershell
+log show --start "2022-04-05 09:00:00" --predicate "subsystem == 'com.microsoft.powershell'"
+```
+
+For more information, run `log show --help` to view the help for the `log show`
+command.
+
+You can also output the log data in JSON format, which allows you to convert
+the event data to PowerShell objects. The following example outputs the events
+in JSON format.The `ConvertFrom-Json` cmdlet is used to convert the JSON data
```
Comment (mikefrobbins) [inline reference/7.4/Microsoft.PowerShell.Core/About/about_Logging_Non-Windows.md:324]:
```suggestion
in JSON format. The `ConvertFrom-Json` cmdlet is used to convert the JSON data
```

File: reference/7.5/Microsoft.PowerShell.Core/About/about_Logging_Non-Windows.md [hunk 4]
```diff
@@ -333,9 +302,38 @@ running, the **process** parameter also accepts a process ID as its value. The
...
+```powershell
+log show --start "2022-04-05 09:00:00" --predicate "subsystem == 'com.microsoft.powershell'"
+```
+
+For more information, run `log show --help` to view the help for the `log show`
+command.
+
+You can also output the log data in JSON format, which allows you to convert
+the event data to PowerShell objects. The following example outputs the events
+in JSON format.The `ConvertFrom-Json` cmdlet is used to convert the JSON data
```
Comment (mikefrobbins) [inline reference/7.5/Microsoft.PowerShell.Core/About/about_Logging_Non-Windows.md:324]:
```suggestion
in JSON format. The `ConvertFrom-Json` cmdlet is used to convert the JSON data
```

**Review by mikefrobbins — DISMISSED** [review]
See my comments.

**Review by michaeltlombardi — DISMISSED** [review]
Looks good, other than the changes Mike requested.

File: reference/7.5/Microsoft.PowerShell.Core/About/about_Logging_Non-Windows.md [hunk 5]
```diff
@@ -279,52 +280,20 @@ log file named `powershell.log`.
...
+events. The PowerShell log events don't appear in the **Console** application
+on macOS. Console app is designed for the older _syslog-based_ logging that
+predates the unified logging system.
 
 ### Viewing PowerShell log data from the command line on macOS
 
 To view PowerShell log data from a command line on macOS, use the `log` command
 in the **Terminal** or other shell host application. These commands can be run
-from **PowerShell**, **Z shell** (**Zsh**), or **Bash**.
+from **PowerShell**, **ZShell**, or **Bash**.
```
Comment (sdwheeler) [inline reference/7.5/Microsoft.PowerShell.Core/About/about_Logging_Non-Windows.md:296]:
```suggestion
from **PowerShell**, **Z Shell**, or **Bash**.
```

**Pull request merged**